src/marqo/core/semi_structured_vespa_index/semi_structured_vespa_schema_template_2_16.sd.jinja2

```diff
@@ -153,6 +153,114 @@ schema {{ index.schema_name }} {
             query(marqo__sort_field_weights_0) tensor<double>(p{})
             query(marqo__sort_field_weights_1) tensor<double>(p{})
             query(marqo__sort_field_weights_2) tensor<double>(p{})
+
+            query(marqo__recency_should_calculate_score): 0
+            query(marqo__recency_should_apply_score): 0
+            query(marqo__recency_decay_function_type): 0
+            query(marqo__recency_timestamp_key) tensor<double>(p{})
+            query(marqo__recency_scale_seconds) double: 604800.0
+            query(marqo__recency_offset_seconds) double: 0.0
+            query(marqo__recency_decay_to) double: 0.5
+        }
+
+        # Recency scoring functions
+        function get_timestamp() {
+            expression {
+                if (count(query(marqo__recency_timestamp_key)) > 0,
+                    reduce(attribute(marqo__score_modifiers) * query(marqo__recency_timestamp_key), sum),
+                    0)
```

Comment:
Does this mean if a documents does not have the `marqo__recency_timestamp_key`, it will be 0, which is considerred as an old documnt? Is this the expected design?